### PR TITLE
Fix Linux TLS Error

### PR DESCRIPTION
**Pull request by HTRamsey**

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
What does `egldeviceintegrations` do? Also is there a need for any other library copying in PostLinkCommon given there are new modules?

File: QGCPostLinkCommon.pri [hunk 1]
```diff
@@ -161,7 +161,10 @@ LinuxBuild {
         position \
         sqldrivers \
         texttospeech \
-        multimedia
+        multimedia \
+        egldeviceintegrations \
+        networkinformation \
+        tls
```
Comment (julianoes) [inline QGCPostLinkCommon.pri:167]: This line fixes the TLS error for me on Linux, nice!

**Comment by HTRamsey** [comment]
@DonLakeFlyer https://doc.qt.io/qt-6/embedded-linux.html, probably not necessary but these three plugins were missing from the qmake build compared to the cmake build. The TLS plugin is the only one actually needed for this issue, so I can change this to just add that one if you want. There are a bunch of other libraries that the qmake build is missing compared to the cmake build but no other ones are needed to make this work.

**Comment by DonLakeFlyer** [comment]
I wonder if the whole embedded Linux thing brings along with it other problems as well. That isn't something we have specifically stated as supported. Only desktop Linux and android.

**Comment by HTRamsey** [comment]
Just to keep it simple and stick to the purpose of this PR I'll just add TLS

**Comment by DonLakeFlyer** [comment]
@HTRamsey Thanks so much for figure this out. It has been plaguing folks for a while.

**Pull request merged**